migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -61,12 +61,12 @@ This section will list the changes that will impact more than one component
 
         </details>
 
-### **_Update icon sizes naming standard_**
+### **_Update to icon sizes naming standard_**
```

Comment:
I think this is worded a bit better

### **_Standardisation of font size naming_**


[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)

-   **Description of change:** Font size classes have been updated to reflect the new size naming convention in the Design System. This update replaces the old naming convention (e.g., xxxl, xxl) with a numerical system (e.g., 3xl, 2xl), similar to the one used for padding and margins.

-   **Reason for change:** This update standardises how sizes are referenced across the system

-   **Components impacted:**

    Any component or layout where you use the xxl or xxxl font-size utility classes.

-   **Migration steps:**

    1. Locate all instances of the font size utility classes in your codebase.
    2. Replace old size name with new corresponding size name: